src/main/java/org/apache/fineract/ReportApi/ReportRequest.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.fineract.ReportApi;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity
+@Data
+@Table(name = "REPORTREQUEST")
+@Getter
+@Setter
+public class ReportRequest {
```

Comment:
Can the validation annotations for checks like not null be put as annotations to the relevant fields in the request DTO itself?

Reply:
Fixed